packages/react-server-cli/src/cli.js

```diff
@@ -1,4 +1,13 @@
-require("babel-core/register");
-
+const fs = require("fs");
+const path = require("path");
 const cli = require(".");
-cli.parseCliArgs().then(cli.run).catch(console.error);
+
+cli.parseCliArgs().then(args => {
+	const {configPath = ''} = args;
```

Comment:
Destructuring with defaults doesn't seem to be supported in node 4.0.0 (at least in my little experiment in the repl just now), so I don't think you'll be able to use it before you require babel.